content/articles/2023/2023-01-15_rust_kernel_module_part2.md

```diff
@@ -3,26 +3,26 @@ Tags: rust, kernel, Linux, ArchLinux
 Language: en
 Summary: Trying to build a hello world out-of-tree Rust kernel module for Linux 6.1. Part two.
 
-In my last [blog posts](/building-an-out-of-tree-rust-kernel-module.html) I
+In my last [blog posts](/building-an-out-of-tree-rust-kernel-module.html), I
 tried to build an out-of-tree kernel module in Rust for the 6.1 stock kernel
 for ArchLinux.
 
-During this process I noticed, that while `CONFIG_HAVE_RUST=y` is set in the
-[ArchLinux kernel config](https://github.com/archlinux/svntogit-packages/blob/706494f4555dca158c9f02932717550e7b66b534/trunk/config)
-, Rust support get's silently disabled, since conflicting options are set.
+During this process, I noticed that while `CONFIG_HAVE_RUST=y` is set in the
+[ArchLinux kernel config](https://github.com/archlinux/svntogit-packages/blob/706494f4555dca158c9f02932717550e7b66b534/trunk/config),
+Rust support get's silently disabled, since conflicting options are set.
```

Comment:
Hmm. Would 
```suggestion
During this process I noticed that, while `CONFIG_HAVE_RUST=y` is set in the
[ArchLinux kernel config](https://github.com/archlinux/svntogit-packages/blob/706494f4555dca158c9f02932717550e7b66b534/trunk/config),
Rust support get's silently disabled, since conflicting options are set.
```
be better? That way it is still a correct sentence if we leave the stuff in between commas away.

But it looks a bit strange to me...

Reply:
Maybe
```suggestion
During this process, I noticed that—while `CONFIG_HAVE_RUST=y` is set in the
[ArchLinux kernel config](https://github.com/archlinux/svntogit-packages/blob/706494f4555dca158c9f02932717550e7b66b534/trunk/config)—Rust support get's silently disabled, since conflicting options are set.
```
?